at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
With à NOTICE I never miss pdu. Debug mode is too wordy.

I think it can help all those who receive SMS, SMS chan_dongle processing is currently incomplete. This is not to debug, but to save the SMS received.

I have already lost SMS because misinterpreted (https://www.diafaan.com/sms-tutorials/gsm-modem-tutorial/online-sms-pdu-decoder/ or http://smstools3.kekekasvi.com/topic.php?id=288), I could not decode them with other tools because I do not not have the PDU.

ps: This will be the only patch that I will apply to the excellent wdoekes/asterisk-chan-dongle O:-)